tools/umi_tools/umi_tools.yml

```diff
@@ -0,0 +1,45 @@
+name: umi_tools
+version: 1.0
+description: Tools for dealing with Unique Molecular Identifiers (UMIs)/Random Molecular Tags (RMTs) and single cell RNA-Seq cell barcodes. 
+keywords:
+    - UMI
+    - RMT
+    - Barcode
+tools:
+    - umi_tools:
+        description: |
+            Tools for dealing with Unique Molecular Identifiers (UMIs)/Random Molecular Tags (RMTs) and single cell RNA-Seq cell barcodes. 
+        homepage: https://github.com/CGATOxford/UMI-tools
+        documentation: https://umi-tools.readthedocs.io/en/latest/
+processes:
+    - dedup:
+        operation: |
+            Set command args to params.umitools_dedup_args
+            The program will execute with the following pattern:
+                umi_tools dedup --log={SAMPLE_ID}.dedup.log {params.umitools_dedup_args} -I {SAMPLE_ID}.bam -S {SAMPLE_ID}.dedup.bam --output-stats={SAMPLE_ID}
+        description: |
+            Groups PCR duplicates and de-duplicates reads to yield one read per group. 
+            Use this when you want to remove the PCR duplicates prior to any downstream analysis.
+        input:
+            - sample_id:
+                type: string
+                description: Sample identifier
+            - bai:
```

Comment:
Not too keen on having an extra element for `bai`  files. I tend to put both in a [list](https://github.com/nf-core/viralrecon/blob/69326f97b1c1d1b8733a0df573cb1f3eff907e6d/main.nf#L908), pass them that way and then use something like [this](https://github.com/nf-core/viralrecon/blob/69326f97b1c1d1b8733a0df573cb1f3eff907e6d/main.nf#L958). Much less syntax to deal with.

Reply:
Agreed, ill get this changed